EIPS/eip-131.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: 131
+title: Blake2b Ethereum Implementation
+author:  Tjaden Hess (@tjade273) , Jay Graber , James Hancock (@madeoftin)
+discussions-to: https://ethereum-magicians.org/t/blake2b-f-precompile/3157/6 https://gitter.im/EIPshepherding/eip-131
+status: Draft
+type: Standards Track
+category: Core
+created: 2019-13-05
+requires: 
+replaces:
```

Comment:
Nitpicking: please remove these two lines.

Reply:
Got it